tools/sgprotocgenopenapi/tools.go

```diff
@@ -0,0 +1,28 @@
+package sgprotocgenopenapi
+
+import (
+	"context"
+	"os/exec"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+// docs:
+// https://github.com/google/gnostic/tree/main/cmd/protoc-gen-openapi
+// https://buf.build/gnostic/gnostic/docs/main:gnostic.openapi.v3
+
+const (
+	version = "0.7.0"
+	name    = "protoc-gen-openapi"
+)
+
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	sg.Deps(ctx, PrepareCommand)
+	return sg.Command(ctx, sg.FromBinDir(name), args...)
+}
+
+func PrepareCommand(ctx context.Context) error {
+	_, err := sgtool.GoInstall(ctx, "github.com/google/gnostic/cmd/"+name, "v"+version)
```

Comment:
@alethenorio actually, @radhus mentioned something ... why don't we do a `go get` followed by `go build` and finally a symlink into .sage/bin?

Here we perform a `go install` which would mess with the binary you might already have on your local system.

Reply:
Why would it mess with your local binary? The GOBIN env variable guarantees that it is built and placed in the right place and not in the Go bin folder

Reply:
Oh wait, the command is only built on line 26 and not executed!
So that means that when the actual `go install` command runs, it runs with the `GOBIN` set, which means it builds into the `GOBIN` dir, which is `.sage/bin`?

Reply:
Correct

Reply:
Nice, sorry for the noise 😅 I don't think there's a problem at all here then, or what do you say @radhus